pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/route/mercury/PostMercuryOrderDummyRoute.java

```diff
@@ -55,9 +56,10 @@ public Object handle(Request request, Response response) {
             log.error("Realm was null for " + mercuryOrderRequest.getRealm());
             return new Result(500, UserErrorMessages.CONTACT_DEVELOPER);
         }
-        publishMessage(mercuryOrderRequest, ddpInstance, userId);
-        return new Result(200);
-
+        String pepperOrderId = publishMessage(mercuryOrderRequest, ddpInstance, userId);
+        JSONObject main = new JSONObject();
+        main.put("PepperOrderId", pepperOrderId);
```

Comment:
I'll channel Simone and say "PepperOrderId" should be a defined constant

Reply:
Thanks fo mentioning that! Fixed